### PR TITLE
Update go-cty to pull in more stdlib funcs.

### PR DESCRIPTION
**Pull request by cpuguy83**
I needed "split" specifically so I can do something like:

```hcl
variable PLATFORMS {
  default = "linux/amd64"
}

target foo {
  platforms = split(",", "${PLATFORMS}")
  # other stuff
}
```

Where the existing "csvdecode" does not work for this because it parses
the string into a list of objects instead of a list of strings.

I went ahead and just added all the available new functions.

Signed-off-by: Brian Goff <cpuguy83@gmail.com>

### CODE REVIEW

File: bake/hcl.go [hunk 1]
```diff
@@ -22,46 +22,76 @@ var (
 		"and":                    stdlib.AndFunc,
 		"byteslen":               stdlib.BytesLenFunc,
 		"bytesslice":             stdlib.BytesSliceFunc,
+		"chomp":                  stdlib.ChompFunc,
+		"chunklist":              stdlib.ChunklistFunc,
+		"ceiling":                stdlib.CeilFunc,
```
Comment (cpuguy83) [inline bake/hcl.go:27]: Should this just be "ceil"?

**Comment by tiborvass** [comment]
I wonder if pathexpand() is also needed, I genuinely don't know. See #269 for a usecase.

File: bake/hcl.go (hunk 1 above)
Comment (tonistiigi) [reply bake/hcl.go:27]: +1

**Comment by cpuguy83** [comment]
I don't see a `PathExpandFunc` in go-cty.

File: bake/hcl.go (hunk 1 above)
Comment (cpuguy83) [reply bake/hcl.go]: Done.

File: bake/hcl.go [hunk 2]
```diff
@@ -22,46 +22,76 @@ var (
...
+		"chunklist":              stdlib.ChunklistFunc,
+		"ceil":                   stdlib.CeilFunc,
 		"csvdecode":              stdlib.CSVDecodeFunc,
 		"coalesce":               stdlib.CoalesceFunc,
+		"coalescelist":           stdlib.CoalesceListFunc,
 		"concat":                 stdlib.ConcatFunc,
+		"contains":               stdlib.ContainsFunc,
+		"distinct":               stdlib.DistinctFunc,
 		"divide":                 stdlib.DivideFunc,
+		"elemment":               stdlib.ElementFunc,
```
Comment (tonistiigi) [inline bake/hcl.go:35]: element

Comment (cpuguy83) [reply bake/hcl.go]: Fixed.

**Pull request merged**